### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sphilee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Contributions to Absolute are welcomed and encouraged! Please see the
 [Contributing to Absolute guide](./contribution/contribution.md)
 
 ### Building Absolute
-./absolut start
+./absolute start
```
Comment (romandev) [inline README.md]: lgtm with nit
I'd like to wrap the command into code block[1]. WDYT?
[1] https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code

**Comment by romandev** [comment]
Also, please update AUTHORS file and then include the change in this patch. :)

File: AUTHORS [hunk 2]
```diff
@@ -19,3 +19,4 @@ Jinho Bang <jinho.bang@samsung.com>
 Taeyun Kim <twhy.kim@gmail.com>
 Umar Faruk <umar97faruk@gmail.com>
 Wuseok Jang <rufia00@gmail.com>
+GwanHyung Lee <gwanhyung787@gmail.com>
```
Comment (romandev) [inline AUTHORS]: Please keep your name in alphabetical order :)

**Comment by sphilee** [comment]
um...  There are questions. How do I delete commit history?

File: AUTHORS [hunk 3]
```diff
@@ -19,3 +19,4 @@ Jinho Bang <jinho.bang@samsung.com>
 Taeyun Kim <twhy.kim@gmail.com>
 Umar Faruk <umar97faruk@gmail.com>
 Wuseok Jang <rufia00@gmail.com>
+Gwanhyung Lee <gwanhyung787@gmail.com>
```
Comment (romandev) [inline AUTHORS]: Please keep your name in alphabetical order.
So, it's better to move your name above Hyung Heo.
```
...
Gwanhyung Lee <gwanhyung787@gmail.com>
Hyung Heo <sunemo83@gmail.com>
...
```

**Comment by romandev** [comment]
@sphilee You can use ```git push``` with -f option as follows:
But I don't think you need to delete history.
When this is merged, your commit history is squashed anyway.

**Pull request merged**